source/_components/media_player.volumio.markdown

```diff
@@ -26,8 +26,17 @@ media_player:
     port: 3000
 ```
 
-Configuration variables:
-
-- **name** (*Optional*): Name of the device
-- **host** (*Required*): IP address or hostname of the device
-- **port** (*Required*): Port number of Volumio service
+{% configuration %}
+name:
+  description: Name of the device
+  required: false
+  type: string
+host:
+  description: IP address or hostname of the device
+  required: true
+  type: string
+port:
```

Comment:
Default value for host is localhost (https://github.com/home-assistant/home-assistant/blob/master/homeassistant/components/media_player/volumio.py#L32)